packages/grafana-ui/src/components/uPlot/hooks.ts

```diff
@@ -94,9 +94,14 @@ export const DEFAULT_PLOT_CONFIG = {
   legend: {
     show: false,
   },
+  gutters: {
+    x: 0,
```

Comment:
you probably don't want this.

with gutter:

![image](https://user-images.githubusercontent.com/43234/98457596-b606d980-214e-11eb-8b57-6937ca11eaa5.png)

without a gutter:

![image](https://user-images.githubusercontent.com/43234/98457620-d46cd500-214e-11eb-937e-099a2bdfc844.png)

Reply:
Grafana's panel content container already has a 8px padding, so the graph needs no extra. Is there a way to not have label ticks on the edge?

Reply:
But I can remove the panel container gutters instead and use the uplot gutters

Reply:
But even with an 8px gutter uPlot renders x-axis outside the graph, think this needs to be fixed in uPlot as Grafana does not know where the axis ticks are rendered so cannot adjust the gutter. Might be solved in Grafana I could set uPlot right-side gutter only,  but looking at this more closely this won't work either as uplot places quite long x-axis ticks on the edge. 

<img width="235" alt="Screen Shot 2020-11-08 at 09 16 30" src="https://user-images.githubusercontent.com/10999/98460266-1d4e7900-21a3-11eb-9b22-0c70053fd41c.png">

<img width="62" alt="Screen Shot 2020-11-08 at 09 07 25" src="https://user-images.githubusercontent.com/10999/98460146-eb88e280-21a1-11eb-81ca-0f87e7298d32.png">

Reply:
uplot issue https://github.com/leeoniya/uPlot/issues/362